project/MimaSettings.scala

```diff
@@ -32,7 +32,18 @@ object MimaSettings {
         exclude[IncompatibleResultTypeProblem]("zio.DurationOps.asScala"),
         exclude[IncompatibleResultTypeProblem]("zio.DurationOps.asScala$extension"),
         exclude[IncompatibleMethTypeProblem]("zio.Queue#Strategy*"),
-        exclude[ReversedMissingMethodProblem]("zio.Queue#Strategy*")
+        exclude[ReversedMissingMethodProblem]("zio.Queue#Strategy*"),
+        exclude[MissingClassProblem]("zio.stream.ZChannel$QRes$"),
+        exclude[MissingClassProblem]("zio.stream.ZChannel$QRes"),
+        exclude[DirectMissingMethodProblem]("zio.stream.ZChannel.mergeAllWith0$default$4"),
+        exclude[DirectMissingMethodProblem]("zio.stream.ZChannel.mergeAllWith0$default$3"),
+        exclude[DirectMissingMethodProblem]("zio.stream.ZChannel.mergeAllWith"),
```

Comment:
This one I find a bit concerning. Is this from the method that accepted `mergeStrategy: MergeStrategy.BackPressure.type` as an argument? Is it possible to keep it, deprecate it, and proxy it to the default `mergeAllWith` (with no param so that it's selected)?

Reply:
Yep sure